frontend/src/components/Users/UserColumns.tsx

```diff
@@ -0,0 +1,50 @@
+import TableActionColumns from "@components/UI/Table/TableActionColumns";
+
+const columns = [
+  {
+    Header: " ",
+    columns: [
+      {
+        Header: "First Name",
+        accessor: "fname",
+      },
+      {
+        Header: "Middle Name",
+        accessor: "mname",
+      },
+      {
+        Header: "Last name",
+        accessor: "lname",
+      },
+      {
+        Header: "Email",
+        accessor: "email",
+      },
+      {
+        Header: "Role",
+        accessor: "role",
+      },
+      {
+        width: 150,
+        Header: "Action",
+        Cell: (data: any) => {
+          let {
+            row: { original },
+          } = data;
+          return (
+            <>
+              <div className="w-full flex flex-end">
+                <TableActionColumns
+                  original={original}
+                />
+              </div>
+            </>

```

Comment:
```suggestion
              <div className="w-full flex flex-end">
                <TableActionColumns
                  original={original}
                />
              </div>
```
remove <></> since you are using <div> </div>